orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/manifest/RunJobManifestContext.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2019 Armory
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.orca.clouddriver.tasks.manifest;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonPOJOBuilder;
+import com.netflix.spinnaker.kork.artifacts.model.Artifact;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import javax.annotation.Nullable;
+import lombok.Builder;
+import lombok.Value;
+
+@Builder(builderClassName = "RunJobManifestContextBuilder", toBuilder = true)
+@JsonDeserialize(builder = RunJobManifestContext.RunJobManifestContextBuilder.class)
+@Value
+public class RunJobManifestContext implements ManifestContext {
+  @Nullable private Map<Object, Object> manifest;
+  @Nullable private List<Map<Object, Object>> manifests;
+
+  private Source source;
+
+  private String manifestArtifactId;
+  private Artifact manifestArtifact;
+  private String manifestArtifactAccount;
+
+  private List<String> requiredArtifactIds;
+  private List<BindArtifact> requiredArtifacts;
+
+  @Builder.Default private boolean skipExpressionEvaluation = false;
+
+  @Override
+  public List<Map<Object, Object>> getManifests() {
+    if (manifest != null) {
+      List<Map<Object, Object>> mfests = new ArrayList<>();
+      mfests.add(manifest);
+      return mfests;
+    }
+    return manifests;
+  }
+
+  @Override
+  public boolean isSkipExpressionEvaluation() {
```

Comment:
Should this return the value of `skipExpressionEvaluation`?  Or if we always want to return `false`, should we remove the `skipExpressionEvaluation` member variable?

Reply:
I think we can always return `false` here since the use case for `skipExpressionEvaluation` is to avoid parsing `${}` in bash scripts. Since this stage only supports single `Job` manifests we can avoid doing that.

Reply:
That makes sense, but in that case would we want to remove the member `skipExpressionEvaluation` from the class as it could be confusing that it can be set but is always ignored.